src/odemis/driver/piezomotor.py

```diff
@@ -0,0 +1,681 @@
+# -*- coding: utf-8 -*-
+'''
+Created on 18 Mar 2020
+
+@author: Philip Winkler
+
+Copyright © 2020, Philip Winkler, Delmic
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the terms
+of the GNU General Public License version 2 as published by the Free Software
+Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+Odemis. If not, see http://www.gnu.org/licenses/.
+'''
+from __future__ import division
+
+import glob
+import logging
+from odemis import model
+from odemis.model import Actuator, CancellableThreadPoolExecutor
+from odemis.util import to_str_escape
+import os
+import threading
+import time
+import serial
+from threading import Thread
+import copy
+from odemis.model import HwError
+from odemis.util.driver import getSerialDriver
+
+BAUDRATE = 115200
+# TODO: Is this always the same for all encoders? Otherwise, it needs to be a parameters in the yaml file.
+WFM_STEPSIZE = 5e-6  # m / step
+USTEPS_PER_WFM = 8192
+ENCODER_STEPSIZE = 1e-9  # m / step
```

Comment:
The encoder stepsize probably needs to be configured manually (run a few wfm steps and get the count, then divide) and put in the yaml file as well. I was hoping the stepsizes are the same for each encoder of the same type, but they probably vary a bit for every model. Maybe we should test this first on the hardware.

Reply:
actually, no, probably that's given by the encoder (need to check which encoder model we're using).